lib/Meta/MetaEntry.php

```diff
@@ -115,7 +124,42 @@ public function getTocs() : array
      */
     public function getDepends() : array
     {
-        return $this->depends;
+        return array_values(array_unique($this->depends));
```

Comment:
Is this needed here because of something upstream that should be fixed? Should the same value exist multiple times?